Task034/maon.py

```diff
@@ -0,0 +1,23 @@
+# Задача 34:  Винни-Пух попросил Вас посмотреть, есть ли в его стихах ритм.
+#  Поскольку разобраться в его кричалках не настолько просто, насколько легко он их придумывает, 
+# Вам стоит написать программу. Винни-Пух считает, что ритм есть, 
+# если число слогов (т.е. число гласных букв) в каждой фразе стихотворения одинаковое.
+#  Фраза может состоять из одного слова, если во фразе несколько слов, то они разделяются дефисами. 
+# Фразы отделяются друг от друга пробелами. Стихотворение  Винни-Пух вбивает в программу с клавиатуры.
+#  В ответе напишите “Парам пам-пам”, если с ритмом все в порядке и “Пам парам”, если с ритмом все не в порядке
+
+# *Пример:*
+
+# **Ввод:** пара-ра-рам рам-пам-папам па-ра-па-да    
+#     **Вывод:** Парам пам-пам 
+
+str = "пара-ра-рам рам-пам-папам па-ра-па-да"
+str = str.split()
+
+def glass_count(phrase):
+    set_gl = {"а", "о", "у", "е", "ы", "я", "и", "ю", "э"}
+    return sum(map(phrase.count, set_gl))
```

Comment:
Наверно лучше обращаться к методe через имя класса, а  а не объекта.
```suggestion
    return sum(map(str.count, set_gl))
```
Но это не сработает, из-за неудачно выбранного имени переменной. Повторяю: плохая идея использовать в качестве имён переменных функций и тд. уже зарезервированные языковые единицы.